logstash-core/lib/logstash/output_delegator.rb

```diff
@@ -8,72 +9,27 @@
 #
 # This plugin also records some basic statistics
 module LogStash class OutputDelegator
-  attr_reader :workers, :config, :worker_count, :threadsafe
+  attr_reader :workers, :config, :threadsafe
 
   # The *args this takes are the same format that a Outputs::Base takes. A list of hashes with parameters in them
   # Internally these just get merged together into a single hash
-  def initialize(logger, klass, default_worker_count, metric, *args)
+  def initialize(logger, klass, default_worker_count, metric, *plugin_args)
     @logger = logger
     @threadsafe = klass.threadsafe?
-    @config = args.reduce({}, :merge)
+    @config = plugin_args.reduce({}, :merge)
     @klass = klass
+    @workers = java.util.concurrent.CopyOnWriteArrayList.new
```

Comment:
I don't see the value since using it is so easy. We no longer support MRI, and I think a wrapper just means a larger area we need to test and more places things can go wrong.